TACC/Core-CMS
# feat: ecep-215 o-float-content--left|right

## Overview

Load a Core-Styles that has `o-float-content--left|right`.

## Related

- [ECEP-215](https://jira.tacc.utexas.edu/browse/ECEP-215)
- requires https://github.com/TACC/Core-Styles/pull/42

## Changes

- load core-styles that has the new class

## Testing

1. Load test pages:
    - https://prod.ecep.tacc.utexas.edu/resources/toolkits-guides/landscape-reports-image-test/ (clears float)
    - https://prod.ecep.tacc.utexas.edu/resources/toolkits-guides/landscape-reports-image/ (does not clear float)
    - https://prod.ecep.tacc.utexas.edu/resources/models-state-change/how-change-state/
2. Verify narrow screen shows full width image.
3. Verify medium screen shows floated image.

## UI

| | 1680px | 992px | 768px |
| - | - | - | - |
| test  1 | ![test 1 1680px](https://user-images.githubusercontent.com/62723358/184014708-dc76730f-76f1-4c45-83f1-4995839109b2.png) | ![test 1 992px](https://user-images.githubusercontent.com/62723358/184014706-f2227708-b7ea-495f-a6c7-f0f1390e8a21.png) | ![test 1 768px](https://user-images.githubusercontent.com/62723358/184014710-92edfe4b-4977-4366-a8c2-a1b1efa5d6e6.png) |
| test 2 | ![test 2 768px](https://user-images.githubusercontent.com/62723358/184014773-a504ef7e-367a-49e4-a7e4-bb3cdd509e59.png) | ![test 2 992px](https://user-images.githubusercontent.com/62723358/184014776-a74938e6-793d-489b-9ded-b25e55b0c40c.png) | ![test 2 1680px](https://user-images.githubusercontent.com/62723358/184014782-6cbea274-321f-44e2-abbe-2854a9681a5b.png) |
